### PR TITLE
ci(repo): Fix integration test runner

### PR DESCRIPTION
**Pull request by dnys1**
Broke due to changes in #2068.

### CODE REVIEW

**Review by Jordan-Nelson — APPROVED** [review]

**Review by HuiSF — APPROVED** [review]

File: .github/workflows/amplify_integration_tests.yaml [hunk 1]
```diff
@@ -30,6 +30,7 @@ jobs:
       - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # 3.1.0
         with:
           persist-credentials: false
+          submodules: true
```
Comment (ragingsquirrel3) [inline .github/workflows/amplify_integration_tests.yaml:33]: question: Is this needed only for android or for other instances of checkout here for other platforms?

**Comment by ragingsquirrel3** [comment]
Does this part need additional commands for submodule, libgit setup? https://github.com/aws-amplify/amplify-flutter/blob/next/.github/composite_actions/install_dependencies/action.yaml#L20

File: .github/workflows/amplify_integration_tests.yaml (hunk 1 above)
Comment (dnys1) [reply .github/workflows/amplify_integration_tests.yaml:33]: It's needed for `aft`

Comment (ragingsquirrel3) [reply .github/workflows/amplify_integration_tests.yaml:33]: Makes sense, but this instance of checkout is only for android and all the other platforms in this file use checkout so was thinking they needed same change (as they also use aft in independent runner/job in matrix).

**Comment by dnys1** [comment]
> Does this part need additional commands for submodule, libgit setup? https://github.com/aws-amplify/amplify-flutter/blob/next/.github/composite_actions/install_dependencies/action.yaml#L20

Ideally, no. Added a check to be safe.

**Review by ragingsquirrel3 — APPROVED** [review]

**Pull request merged**